common/app/model/pageElements/PageElement.scala

```diff
@@ -0,0 +1,195 @@
+package model.pageElements
```

Comment:
should this maybe have a more specific name if it's only gonna be used in dotcom rendering?

Reply:
Yep that seems fine